### ISSUE TITLE
HLS: Duration not displayed before playing starts

### ISSUE THREAD

**Issue by ile** [sampotts/plyr#701]
Duration won't be displayed even when custom duration is set (it will be shown as 00:00). Only when playing starts the real duration will be shown (from loadedmetadata).

I think duration should be shown even when it hasn't been played/started (if the custom duration is set).

### Expected behaviour 
Duration should be shown (imo).

Players affected:
- [x] HTML5 Video (HLS)

Using the latest version at CDN.

### Steps to reproduce 
```
plyr.setup(video_element, { duration: 99999, disableContextMenu: false, displayDuration: true, controls: ['play-large', 'play', 'progress', 'current-time', 'duration', 'mute', 'volume', 'captions', 'fullscreen'] })
```

**Comment by ile** [comment]
Also when the playing starts, it would be OK if the real duration was shown at that point (from loadedmetadata).

`preload` doesn't seem to happen, so I need to show the custom duration.

**Comment by ile** [comment]
Actually custom duration is shown after playing happens, so the title of this issue isn't quite accurate.

But as said, I would like it to be shown before the play happens.

**Comment by friday** [comment]
Confirmed: https://jsfiddle.net/v0u5e4us/1/

Doesn't seem to happen without `invertTime: false`, or in vanilla html5 or vimeo. Youtube has a related but not identical issue which is fixed but not released.

**Comment by friday** [comment]
Fixed in v3.3.10
